app/src/main/java/com/github/se/polyfit/ui/screen/LoginScreen.kt

```diff
@@ -58,54 +59,67 @@ fun LoginScreen(goTo: () -> Unit) {
       modifier = Modifier.fillMaxSize().testTag("LoginScreen"),
       color = MaterialTheme.colorScheme.background) {
         Column(
+            modifier = Modifier.padding(24.dp),
             verticalArrangement = Arrangement.Center,
             horizontalAlignment = Alignment.CenterHorizontally) {
-              Spacer(Modifier.weight(0.2f))
-
-              Title(Modifier.testTag("LoginTitle"), 50.sp)
-
-              Spacer(Modifier.weight(0.6f))
-
-              SignInButton {
-                Log.d("LoginScreen", "button clicked")
-                createSignInIntent()
-              }
-
-              Spacer(Modifier.weight(0.03f))
+              Spacer(Modifier.weight(0.15f))
 
               Text(
-                  termText, textAlign = TextAlign.Center, modifier = Modifier.testTag("LoginTerms"))
-
-              Spacer(Modifier.weight(0.2f))
+                  text = "Welcome",
```

Comment:
The authentication mechanism seems to be malfunctioning. Have you tested it on your end to see if it works?

Reply:
the authetication is indeed malfunctionning but I didn't implement it. I can take off this code and replace it with the functionning one if there is one